src/core/linalg/src/sparse/4C_linalg_vector.hpp

```diff
@@ -212,24 +212,20 @@ namespace Core::LinAlg
       return vector_->Multiply(TransA, TransB, ScalarAB, A, B, ScalarThis);
     }
 
-    //! Puts element-wise reciprocal values of input Multi-vector in target.
-    int reciprocal(const Epetra_MultiVector& A) { return vector_->Reciprocal(A); }
-
     //! Multiply a Core::LinAlg::MultiVector<double> with another, element-by-element.
     int multiply(double ScalarAB, const Epetra_MultiVector& A, const Epetra_MultiVector& B,
         double ScalarThis)
     {
       return vector_->Multiply(ScalarAB, A, B, ScalarThis);
     }
 
-    int replace_global_values(int NumEntries, const double* Values, const int* Indices)
-    {
-      return vector_->ReplaceGlobalValues(NumEntries, Values, Indices);
-    }
+    //! Puts element-wise reciprocal values of input Multi-vector in target.
+    int reciprocal(const Epetra_MultiVector& A) { return vector_->Reciprocal(A); }
```

Comment:
This public method asks for an `Epetra_MultiVector`, so it kind of exposes the `Epetra_MultiVector` wrapped in this class to the outside world. Shouldn't it take a `LINALG::Vector` as input?